pkg/composableschemadsl/compiler/compiler_test.go

```diff
@@ -1254,17 +1254,23 @@ func TestCompile(t *testing.T) {
 			},
 		},
 		{
-			"duplicate use pragmas",
```

Comment:
Got rid of this test, since we don't want to enforce this anymore; added a test from `schemadsl` that captured the wildcard + expiration issue.